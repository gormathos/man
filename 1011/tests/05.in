sol_id;user_id;task_id;score
179862;1031;2004;14.63
179869;1008;2001;80.65
179877;1001;2001;41.94
179883;1019;2001;41.94
179887;1021;2001;6.45
179890;1022;2003;95.65
179890;1008;2004;29.27
179899;1005;2001;48.39
179903;1011;2003;26.09
179904;1026;2003;19.57
179912;1007;2003;39.13
179914;1022;2003;43.48
179922;1013;2002;4.08
179922;1016;2003;78.26
179922;1010;2003;17.39
179931;1003;2003;50.00
179931;1010;2001;3.23
179934;1020;2003;19.57
179943;1027;2003;56.52
179950;1007;2002;10.20
179960;1011;2004;36.59
179963;1005;2004;34.15
179965;1014;2004;4.88
179972;1029;2001;9.68
179972;1027;2001;6.45
179978;1018;2004;12.20
179983;1027;2001;19.35
179986;1029;2003;13.04
179988;1027;2001;22.58
179988;1015;2001;45.16
179992;1019;2002;14.29
179994;1006;2003;2.17
180003;1010;2001;0.00
180003;1003;2001;77.42
180008;1025;2003;80.43
180017;1002;2003;50.00
180017;1015;2002;4.08
180017;1005;2002;8.16
180026;1017;2003;52.17
180027;1017;2003;30.43
180028;1007;2004;4.88
180037;1021;2002;65.31
180043;1028;2004;65.85
180044;1012;2004;51.22
180048;1009;2002;26.53
180058;1012;2002;71.43
180060;1014;2001;96.77
180061;1025;2002;28.57
180067;1030;2004;2.44
180072;1017;2004;0.00
180072;1023;2001;32.26
180077;1002;2004;26.83
180080;1010;2004;85.37
180090;1004;2001;67.74
180097;1001;2002;67.35
180102;1011;2001;3.23
180108;1030;2001;3.23
180118;1030;2002;4.08
180118;1031;2001;22.58
180128;1020;2002;55.10
180133;1017;2002;30.61
180140;1003;2004;58.54
180141;1001;2003;10.87
180148;1021;2002;65.31
180154;1029;2001;90.32
180161;1006;2001;9.68
180170;1030;2002;4.08
180180;1021;2003;6.52
180187;1017;2002;0.00
180195;1016;2001;32.26
180205;1027;2004;87.80
180209;1026;2002;16.33
180214;1016;2003;76.09
180222;1015;2003;47.83
180231;1005;2003;97.83
180232;1028;2003;47.83
180236;1012;2004;80.49
180239;1011;2004;26.83
180246;1032;2004;14.63
180248;1005;2003;28.26
180248;1012;2001;6.45
180249;1005;2004;46.34
180254;1002;2003;30.43
180264;1028;2004;95.12
180267;1024;2004;0.00
180272;1001;2001;29.03
180277;1021;2001;51.61
180279;1030;2003;8.70
180289;1029;2002;53.06
180299;1026;2004;4.88
180299;1006;2003;2.17
180306;1017;2003;95.65
180316;1002;2002;83.67
180321;1002;2004;9.76
180324;1023;2003;2.17
180327;1027;2004;12.20
180328;1031;2002;12.24
180329;1004;2003;39.13
180329;1024;2004;70.73
180338;1002;2003;50.00
180341;1007;2002;36.73
180345;1008;2004;2.44
180346;1002;2001;80.65
180346;1015;2004;0.00
180350;1020;2004;90.24
180350;1021;2004;87.80
180356;1003;2002;4.08
180360;1019;2004;24.39
180361;1019;2002;8.16
180362;1011;2002;6.12
180366;1030;2004;53.66
180371;1004;2003;50.00
180376;1025;2002;6.12
180378;1032;2003;34.78
180379;1024;2003;97.83
180380;1023;2001;25.81
180386;1003;2001;67.74
180393;1031;2003;13.04
180399;1017;2001;16.13
180399;1010;2002;10.20
180399;1011;2002;0.00
180409;1018;2001;32.26
180413;1009;2004;56.10
180417;1026;2003;60.87
180418;1013;2003;58.70
180424;1029;2001;54.84
180432;1028;2001;9.68
180440;1003;2002;69.39
180448;1016;2003;50.00
180451;1018;2004;7.32
180455;1020;2002;18.37
180460;1014;2003;54.35
180466;1015;2004;14.63
180470;1016;2002;36.73
180474;1029;2003;73.91
180476;1022;2003;30.43
180477;1020;2001;6.45
180483;1016;2001;3.23
180491;1007;2001;35.48
180493;1012;2004;51.22
180495;1026;2002;57.14
180501;1019;2003;2.17
180506;1001;2004;21.95
180510;1022;2004;0.00
180515;1032;2001;0.00
180522;1013;2001;35.48
180528;1013;2001;58.06
180532;1030;2004;29.27
180535;1012;2002;63.27
180535;1018;2002;30.61
180541;1031;2003;56.52
180541;1031;2002;10.20
180549;1006;2002;30.61
180558;1024;2002;28.57
180566;1002;2004;82.93
180566;1020;2003;60.87
180566;1017;2004;46.34
180576;1013;2002;28.57
180578;1031;2001;70.97
180583;1006;2002;12.24
180589;1022;2002;32.65
180599;1015;2001;19.35
180608;1032;2004;0.00
180617;1009;2003;43.48
180627;1003;2003;6.52
180630;1003;2002;2.04
180633;1031;2002;55.10
180641;1032;2002;93.88
180645;1015;2004;53.66
180648;1029;2003;0.00
180657;1005;2002;18.37
180663;1018;2001;25.81
180668;1010;2004;24.39
180668;1010;2003;26.09
180668;1032;2002;10.20
180676;1018;2002;73.47
180677;1004;2003;17.39
180685;1026;2001;3.23
180686;1011;2001;0.00
180694;1032;2001;6.45
180703;1013;2002;61.22
180705;1016;2004;39.02
180709;1004;2002;14.29
180717;1018;2003;58.70
180718;1008;2003;43.48
180725;1001;2002;71.43
180732;1008;2002;0.00
180732;1009;2003;13.04
180733;1008;2001;90.32
180738;1007;2003;17.39
180747;1010;2001;58.06
180749;1029;2004;0.00
180759;1023;2004;90.24
180761;1024;2002;0.00
180765;1007;2001;83.87
180772;1028;2002;8.16
180774;1012;2003;43.48
180775;1004;2004;0.00
180776;1029;2002;20.41
180780;1031;2003;43.48
180784;1013;2003;54.35
180789;1006;2004;95.12
180790;1022;2004;31.71
180798;1017;2004;21.95
180798;1015;2002;10.20
180800;1006;2003;56.52
180805;1027;2002;8.16
180815;1027;2004;9.76
180818;1007;2004;21.95
180820;1009;2002;28.57
180822;1005;2002;4.08
180829;1025;2002;16.33
180834;1025;2001;45.16
180839;1014;2002;6.12
180841;1002;2002;20.41
180841;1001;2002;75.51
180841;1014;2001;32.26
180842;1018;2003;10.87
180849;1004;2002;36.73
180859;1028;2003;82.61
180862;1001;2004;48.78
180866;1025;2004;12.20
180868;1022;2001;6.45
180876;1024;2002;4.08
180878;1032;2003;21.74
180880;1017;2002;36.73
180887;1022;2001;3.23
180887;1026;2001;61.29
180897;1021;2004;19.51
180903;1032;2004;73.17
180905;1002;2001;58.06
180911;1004;2001;25.81
180918;1023;2003;60.87
180928;1001;2001;6.45
180930;1017;2001;9.68
180938;1023;2004;60.98
180939;1014;2003;52.17
180945;1006;2004;34.15
180949;1025;2001;12.90
180959;1009;2001;3.23
180962;1021;2004;56.10
180968;1023;2004;26.83
180974;1016;2002;34.69
180980;1011;2001;19.35
180987;1031;2001;12.90
180996;1026;2003;36.96
180996;1019;2003;36.96
181004;1028;2003;17.39
181008;1024;2004;34.15
181011;1026;2004;4.88
181013;1028;2001;22.58
181018;1029;2004;9.76
181019;1029;2004;100.00
181024;1005;2003;13.04
181025;1014;2001;16.13
181035;1009;2001;93.55
181036;1007;2001;6.45
181043;1021;2003;50.00
181052;1025;2001;41.94
181056;1030;2001;61.29
181056;1006;2001;16.13
181060;1027;2002;18.37
181066;1021;2003;2.17
181071;1001;2003;73.91
181076;1005;2004;0.00
181086;1021;2002;10.20
181096;1026;2004;7.32
181105;1009;2004;95.12
181107;1024;2003;65.22
181113;1015;2002;4.08
181118;1023;2001;38.71
181119;1009;2003;43.48
181128;1008;2001;16.13
181131;1009;2004;2.44
181140;1003;2001;0.00
181148;1008;2004;9.76
181148;1005;2001;6.45
181154;1015;2001;87.10
181156;1020;2001;83.87
181161;1012;2003;15.22
181166;1020;2004;75.61
181174;1008;2003;32.61
181176;1023;2002;53.06
181182;1013;2004;36.59
181189;1020;2001;45.16
181190;1030;2002;55.10
181191;1011;2003;23.91
181196;1023;2002;12.24
181200;1014;2004;14.63
181203;1004;2004;41.46
181208;1010;2002;2.04
181218;1008;2002;73.47
181218;1024;2001;0.00
181219;1007;2003;0.00
181219;1012;2002;2.04
181227;1024;2001;9.68
181228;1010;2004;80.49
181233;1011;2004;24.39
181238;1012;2003;36.96
181248;1019;2004;36.59
181254;1010;2003;8.70
181262;1016;2001;90.32
181266;1005;2001;51.61
181267;1004;2002;16.33
181269;1008;2002;0.00
181269;1019;2003;63.04
181277;1014;2002;59.18
181277;1028;2002;8.16
181282;1017;2001;77.42
181282;1030;2001;80.65
181287;1027;2003;2.17
181289;1002;2002;53.06
181295;1018;2001;35.48
181298;1028;2002;28.57
181307;1003;2004;2.44
181308;1025;2003;34.78
181314;1026;2001;25.81
181324;1013;2004;7.32
181330;1029;2002;2.04
181335;1013;2004;75.61
181342;1022;2002;28.57
181349;1003;2004;29.27
181358;1028;2004;65.85
181361;1031;2004;65.85
181365;1012;2001;6.45
181372;1013;2003;30.43
181378;1010;2002;34.69
181383;1030;2003;8.70
181388;1032;2001;54.84
181394;1013;2001;12.90
181397;1020;2003;69.57
181400;1020;2004;17.07
181403;1016;2004;85.37
181404;1027;2002;67.35
181410;1015;2003;43.48
181415;1003;2003;26.09
181421;1014;2003;34.78
181422;1014;2004;2.44
181422;1012;2001;70.97
181423;1004;2001;6.45
181425;1019;2002;4.08
181430;1006;2004;100.00
181432;1023;2003;39.13
181434;1022;2004;68.29
181436;1024;2003;71.74
181442;1028;2001;9.68
181447;1002;2001;19.35
181456;1018;2003;69.57
181461;1018;2002;44.90
181467;1018;2004;58.54
181467;1025;2003;8.70
181469;1024;2001;25.81
181470;1009;2002;8.16
181476;1025;2004;85.37
181481;1032;2003;10.87
181488;1008;2003;43.48
181498;1011;2002;61.22
181501;1025;2004;0.00
181511;1022;2002;2.04
181516;1006;2002;44.90
181524;1022;2001;29.03
181527;1016;2002;6.12
181537;1014;2002;8.16
181541;1006;2001;3.23
181547;1019;2001;6.45
181556;1015;2003;0.00
181558;1004;2004;53.66
181559;1020;2002;10.20
181563;1031;2004;36.59
181571;1009;2001;16.13
181571;1030;2003;84.78
181573;1026;2002;61.22
181581;1007;2002;18.37
181588;1032;2002;18.37
181588;1027;2003;97.83
181590;1007;2004;46.34
181591;1011;2003;0.00
181599;1016;2004;39.02
181604;1021;2001;58.06
181605;1001;2004;43.90
181615;1019;2001;74.19
181621;1023;2002;4.08
181628;1001;2003;91.30
181635;1019;2004;0.00
