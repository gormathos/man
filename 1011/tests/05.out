1001: Very Good [64.38%]
1002: Excellent [74.31%]
1003: Very Good [63.84%]
1004: Participant [52.03%]
1005: Participant [53.54%]
1006: Participant [54.39%]
1007: Participant [51.52%]
1008: Participant [59.13%]
1009: Very Good [65.18%]
1010: Participant [51.05%]
1011: Participant [35.81%]
1012: Very Good [66.59%]
1013: Very Good [63.40%]
1014: Participant [56.23%]
1015: Participant [49.70%]
1016: Very Good [72.67%]
1017: Very Good [64.03%]
1018: Participant [59.26%]
1019: Participant [47.03%]
1020: Excellent [74.69%]
1021: Very Good [65.29%]
1022: Participant [56.41%]
1023: Very Good [60.72%]
1024: Participant [55.73%]
1025: Participant [59.88%]
1026: Participant [47.67%]
1027: Very Good [68.89%]
1028: Participant [57.22%]
1029: Excellent [79.32%]
1030: Very Good [68.55%]
1031: Very Good [62.11%]
1032: Very Good [64.17%]
